src/App/integrationTests/managementRegime/App.managementRegimeCreateOffline.test.js

```diff
@@ -87,11 +87,7 @@ describe('Offline', () => {
 
     await saveMR()
 
-    expect(
-      await screen.findByText(
-        'The management regime has been saved on your computer and in the MERMAID online system.',
-      ),
-    )
+    expect(await screen.findByText('The management regime has been saved on your computer.'))
```

Comment:
is it worth also mentioning that it will attempt to save on the mermaid db once tehre's a steady internet connection?

Reply:
Probably, but Ill leave wording to @alanjleonard. He has a task to look through error messages and improve language as needed.

(I like that idea FWIW)